db/model/model.js

```diff
@@ -79,3 +79,27 @@ exports.updateCommentsById = (article_id, request) => {
       return rows[0];
     });
 };
+
+exports.updateArticle = (article_id, votesInc) => {
+  const votes = votesInc.inc_votes;
+  const inputs = [votes, article_id];
+  console.log(inputs);
+  return db
+    .query(
+      `UPDATE articles
+    SET votes = votes + $1 
+    WHERE article_id = $2`,
+      inputs
+    )
+    .then(() => {
```

Comment:
This check should be completed before you try and update the article. Consider the check exists function that was proposed in the error handling lecture.